### ISSUE TITLE
Formatting all elm files in a directory

### ISSUE THREAD

**Issue by avh4** [avh4/elm-format#16]
`elm-format src` (`src` is a directory) should format all `*.elm` files found (recursively) in the given directory.

Using `--output FILE` with a directory should exit with an error message.

The following prompt should be given (unless `--yes` is specified):

```
This will overwrite the following 4 files to use Elm’s preferred style:

  src/TodoMVC.elm
  src/TodoMVC/Model.elm
  src/TodoMVC/Update.elm
  src/TodoMVC/View.elm

This cannot be undone! Make sure to back up these files before proceeding.

Are you sure you want to overwrite these files with formatted versions? (y/N)
```

**Comment by Janiczek** [comment]
What should be the behaviour when you have eg. 3 `.elm` files and the first one has compiler errors in it?
- print something, exitFailure (leaving the remaining files as are)
- print something, and continue parsing the rest of the files?

**Comment by avh4** [comment]
I'd say do the simplest option that seems technically correct to you, and open a new issue to discuss whether we want to handle it in a more complicated way.

**Comment by avh4** [comment]
Fixed in #41
